common/lib/xmodule/xmodule/tests/test_course_module.py

```diff
@@ -277,3 +277,60 @@ class DiscussionTopicsTestCase(unittest.TestCase):
     def test_default_discussion_topics(self):
         d = get_dummy_course('2012-12-02T12:00')
         self.assertEqual({'General': {'id': 'i4x-test_org-test_course-course-test'}}, d.discussion_topics)
+
+
+class TeamsConfigurationTestCase(unittest.TestCase):
+    """
+    Tests for the configuration of teams and the helper methods for accessing them.
+    """
+    counter = 0
+
+    def setUp(self):
+        super(TeamsConfigurationTestCase, self).setUp()
+        self.course = get_dummy_course('2012-12-02T12:00')
+
+    def add_team_configuration(self, max_team_size=3, topics=None):
+        """ Add a team configuration to the course. """
+        if topics is None:
+            topics = []
+        self.course.teams_configuration = {"max_team_size": max_team_size, "topics": topics}
+
+    def make_topic(self):
+        """ Make a sample topic dictionary. """
+        topic_id = "topic_id_{}".format(self.counter)
+        self.counter = self.counter + 1
+        return {"display_name": "Display Name", "description": "Description", "id": topic_id}
+
+    def test_teams_enabled_no_teams(self):
+        # Make sure we can detect when no teams exist.
+        self.assertFalse(self.course.teams_enabled)
+
+    def test_teams_enabled_no_topics(self):
+        self.add_team_configuration(4)
+        self.assertFalse(self.course.teams_enabled)
+
+    def test_teams_enabled_with_teams(self):
+        self.add_team_configuration(4, topics=[self.make_topic(), self.make_topic()])
+        self.assertTrue(self.course.teams_enabled)
+
+    def test_teams_max_size_no_teams(self):
```

Comment:
I'm not sure what's meant by `no_teams` in this context. Teams don't exist in this teams configuration - just topics and max sizes. Maybe this could be `test_teams_max_size_no_teams_configuration`?